tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -1379,12 +1434,28 @@ private long[] setBestNextAndSameVoteValues(Set<T> sortedValues, HashMap<T, Long
                 winningValue = value;
                 weightArray[0] = valueWeight;
                 valuesWithSameVotes.add(value);
+                annotateTranscript("The winning value (O) is '%s', with a weight of %d", winningValue, valueWeight);
+                if (sortedValues.size() == 1) {
+                    annotateTranscript("- No other values received votes."); // uncontested
+                }
             } else {
                 if (i == 1) {
                     // get the next item if there is one
                     if (iterator.hasNext()) {
                         nValue = value;
                         weightArray[1] = valueWeight;
+                        // sortedValues.size() >= 2 - explain why O won and N lost.
+                        // We have to perform the function of the votesThenUcaCollator one more time
+                        if (weightArray[0] > weightArray[1]) {
+                            annotateTranscript("- This is the winning value because it has the highest weight (voting score).");
+                        } else if(winningValue.equals(baselineValue)) {
+                            annotateTranscript("- This is the winning value because it is the same as the baseline value, though the weight was otherwise equal to the next-best."); // aka blue star
+                        } else if(winningValue.equals(CldrUtility.INHERITANCE_MARKER)) {
+                            annotateTranscript("- This is the winning value because it is the inheritance marker, though the weight was otherwise equal to the next-best."); // triple up arrow
+                        } else {
+                            annotateTranscript("- This is the winning value because it comes earlier than '%s' when the text was sorted, though the weight was otherwise equal to the next-best.", nValue);
+                        }
+                        annotateTranscript("The Next-best (N) value is '%s', with weight %d", nValue, valueWeight);
```

Comment:
these 12 new lines could all go in a subroutine

Reply:
a subroutine next to votesThenUcaCollator